client/lib/abtest/active-tests.js

```diff
@@ -45,6 +45,15 @@ module.exports = {
 		},
 		defaultVariation: 'original'
 	},
+	sidebarPlanLink: {
```

Comment:
I would anticipate we may have a number of tests against the sidebar plan link. Can we make this test name more specific about what we're testing, like `sidebarPlansMyPlan`?